packages/neos-ui/src/Sagas/CR/ContentDimensions/index.js

```diff
@@ -1,66 +1,99 @@
-import {takeLatest} from 'redux-saga';
-import {put, select, race, take} from 'redux-saga/effects';
+import {put, select, race, take, call} from 'redux-saga/effects';
 import {$get} from 'plow-js';
 
 import {actions, actionTypes, selectors} from '@neos-project/neos-ui-redux-store';
 import backend from '@neos-project/neos-ui-backend-connector';
-function * watchSelectPreset() {
-    // Only process the latest selection and cancel previous ones
-    yield * takeLatest(actionTypes.CR.ContentDimensions.SELECT_PRESET, updateContentCanvasSrc);
+
+function * watchSelectPreset({configuration}) {
```

Comment:
add a comment here explaining in 1-2 sentences what the saga does, and especially the idea why we need to forget all nodes on dimension switch and reload the Node List.

Reply:
Actually we don't. I thought it would be a good opportunity to get rid of a few nodes to not let the node state grow too large. But after thinking about it, this might not be the correct place to handle this problem, so I'm going to remove it.